.github/workflows/well-known-environment.yml

```diff
@@ -142,29 +146,14 @@ jobs:
           branch: ${{ env.branch }}
         shell: zsh {0}
 
-      - name: Configure AWS credentials for ECS (dev)
-        uses: aws-actions/configure-aws-credentials@v4
-        if: startsWith(env.branch, 'env/dev/')
+      - name: Configure AWS credentials for account
+        uses: ./.github/actions/configure-aws-credentials
         with:
-          role-to-assume: ${{ secrets.UHD_ECS_ROLE_DEV }}
-          aws-region: ${{ env.AWS_REGION }}
-          role-chaining: true
-
-      - name: Configure AWS credentials for ECS (test)
-        uses: aws-actions/configure-aws-credentials@v4
-        if: startsWith(env.branch, 'env/test/')
-        with:
-          role-to-assume: ${{ secrets.UHD_ECS_ROLE_TEST }}
-          aws-region: ${{ env.AWS_REGION }}
-          role-chaining: true
-
-      - name: Configure AWS credentials for ECS (uat)
-        uses: aws-actions/configure-aws-credentials@v4
-        if: startsWith(env.branch, 'env/uat/')
-        with:
-          role-to-assume: ${{ secrets.UHD_ECS_ROLE_UAT }}
-          aws-region: ${{ env.AWS_REGION }}
-          role-chaining: true
+          environment_name: ${{ env.branch }}
```

Comment:
I think `env.branch` will resolve to something like `env/dev`, so you'll need some string manipulation here to get the actual env name

Reply:
That doesn't matter, the composite action takes the input `environment_name` and does a substring search i.e. `${{ contains(inputs.environment_name, 'dev') }}`
i.e. if `dev` is a substring of the search term then it will return true